static/app/components/sidebar/sidebarPanelItem.tsx

```diff
@@ -3,58 +3,33 @@ import styled from '@emotion/styled';
 import ExternalLink from 'sentry/components/links/externalLink';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
+import type {Broadcast} from 'sentry/types/system';
 import {trackAnalytics} from 'sentry/utils/analytics';
 import useOrganization from 'sentry/utils/useOrganization';
 
-type Props = {
+interface SidebarPanelItemProps
+  extends Partial<
+    Pick<Broadcast, 'link' | 'category' | 'title' | 'hasSeen' | 'message'>
+  > {
   /**
    * Content rendered instead the panel item
    */
   children?: React.ReactNode;
-  /**
-   * The text for the CTA link at the bottom of the panel item
-   */
-  cta?: string;
-  /**
-   * Has the item been seen? affects the styling of the panel item
-   */
-  hasSeen?: boolean;
-  /**
-   * The URL to use for the CTA
-   */
-  link?: string;
-  /**
-   * A message with muted styling which appears above the children content
-   */
-  message?: React.ReactNode;
-  /**
-   * The title of the sidebar item
-   */
-  title?: string;
-  /**
-   * Actions to the right of the title
-   */
-  titleAction?: React.ReactNode;
-};
+  ctaText?: string;
+}
 
 function SidebarPanelItem({
```

Comment:
is it worth making any changes to `SidebarPanelItem` since that's the old experience?

Reply:
There was a minor improvement in the types, as I noticed we could reuse the ones defined for Broadcast.

As for this component, even though it won't be used in the broadcast experience, it is still being used in forhttps://github.com/getsentry/sentry/blob/3634c69f7ff8b55114f8b69d0ced5b4b96b1077d/static/app/components/sidebar/serviceIncidents.tsx#L60